ts/server.go

```diff
@@ -89,8 +89,9 @@ func (s *Server) handleQuery(w http.ResponseWriter, r *http.Request, _ httproute
 		response.Results = append(response.Results, &TimeSeriesQueryResponse_Result{
 			Name:             q.Name,
 			Sources:          sources,
-			SourceAggregator: q.SourceAggregator,
-			Downsampler:      q.Downsampler,
+			SourceAggregator: q.GetSourceAggregator().Enum(),
```

Comment:
Ah, they're special cased in timeseries.proto. Feel free to leave it as-is, I'll fix it in a follow-up change.